File: w2/home/arrayScan()/arrayScan().c
#include <stdio.h>

int arrayScan(FILE *in, int array[], int limit) {
    int len = 0;
    
    for ( ; len < limit && fscanf(in, "%d", &array[len]) == 1; len++ );
    return len;
}


int main() {
    FILE *in = fopen("task.in", "r");
    int size = 10;
    int array[size];
    int limit = 5;
    printf("%d\n", arrayScan(in, array, limit));


    return 0;
}

File: w4/php_dz/arrayScan()/arrayScan().php
<?php
    function arrayScan(&$in, &$array, $limit) {
        $array = explode(' ', fread($in, $limit * 2));
    }

    $in = fopen("task.in", "r");

    $array = array();
    arrayScan($in, $array, 7);

    var_dump($array);

    fclose($in);
?>
